src/vs/editor/contrib/codeAction/codeActionModel.ts

```diff
@@ -72,55 +68,25 @@ export class CodeActionOracle {
 		return undefined;
 	}
 
-	private _getRangeOfSelectionUnlessWhitespaceEnclosed(): Selection {
```

Comment:
Yeah, I remember we had that for a very specific reason... Needs some deeper thinking but I believe it was to prevent to lightbulb from showing 'randomly' while clicking into the void/whitespace....

Reply:
Ok, I'll restore it